### ISSUE TITLE
Add modpe function to add a custom status effect

### ISSUE THREAD

**Issue by connerfick** [zhuowei/MCPELauncher#857]
I would like a function that creates a new status effect, rather than giving the player an already existing one.

**Comment by officialmcpro32** [comment]
@zhuowei this is a good idea

**Comment by DiegoGTRatty** [comment]
So this would basically be like:

```
function newEffect(vars :v) {
//"pushes" a new effect ID or var into MCPE
if (vars == "moddedEffect") {
//insert what the effect does here. :v
}
}
```

**Comment by connerfick** [comment]
I like this better.

[New-Status-Effects.txt](https://github.com/zhuowei/MCPELauncher/files/207295/New-Status-Effects.txt)

**Comment by DiegoGTRatty** [comment]
@connerfick 
Maybe add a true/false variable on whether the effect changes when more power is used.

`ModPE.setEffect(id, "name", "image", false);`

Also, if the effect time is out, how will the flying status be set back to normal?

**Comment by connerfick** [comment]
@DiegoGTRatty

Good idea making more power add more abilities.  As for reset flight effect we could add a fifth variable that resets the function when true.

**Comment by connerfick** [comment]
we could also add a setting that defines a particles image, but you would have to be able leave it blank(because I don't like the particles at all)

**Comment by DiegoGTRatty** [comment]
Now, I'M GOING IN FOR THE REBOUND!

@zhuowei See this, read this, and just do it. :v

**Comment by connerfick** [comment]
@zhuowei

**Comment by ISAAC300** [comment]
@zhuowei dont know how add usefull functions ;-;
